tests/rules/javascript/test.deprecated_entities.js

```diff
@@ -0,0 +1,77 @@
+import { VALIDATION_WARNING, DEPRECATED_ENTITIES } from 'const';
+import { singleLineString } from 'utils';
+import JavaScriptScanner from 'scanners/javascript';
+
+describe('no_document_write', () => {
+  for (let entity of DEPRECATED_ENTITIES) {
+    let obj = entity.object;
```

Comment:
I dislike short variable names like this; they're especially hard to read for non-native English speakers, but also we don't need to use them; we have space for `object`, `property`, etc.

Though in this instance we shouldn't really need them at all, you're only using them twice and using as many lines to define the variable.